src/Dapr.Actors/Client/ActorProxy.cs

```diff
@@ -21,6 +21,12 @@ namespace Dapr.Actors.Client
     public class ActorProxy : IActorProxy
     {
         internal static readonly ActorProxyFactory DefaultProxyFactory = new ActorProxyFactory();
+
+        /// <summary>
+        /// The <see cref="System.Text.Json.JsonSerializerOptions"/> used for outbound actor message serialization
+        /// </summary>
+        public static readonly JsonSerializerOptions JsonSerializerOptions = new JsonSerializerOptions();
```

Comment:
We should find another way to accomplish this. Using a static for configuration will result in a lot of problems down the road - it's not testable and it's easy for libraries to use in a way that doesn't compose.